content/docs/csidriver/upgradation/drivers/unity.md

```diff
@@ -20,24 +20,20 @@ You can upgrade the CSI Driver for Dell Unity XT using Helm or Dell CSI Operator
 
 Preparing myvalues.yaml is the same as explained in the install section.
 
-To upgrade the driver from csi-unity v2.6.0 to csi-unity v2.7.0
+To upgrade the driver from csi-unity v2.7.0 to csi-unity v2.8.0
 
-1. Get the latest csi-unity v2.7.0 code from Github using `git clone -b v2.7.0 https://github.com/dell/csi-unity.git`.
+1. Get the latest csi-unity v2.8.0 code from Github using `git clone -b v2.8.0 https://github.com/dell/csi-unity.git`.
 2. Copy the helm/csi-unity/values.yaml to the new location csi-unity/dell-csi-helm-installer and rename it to myvalues.yaml. Customize settings for installation by editing myvalues.yaml as needed.
 3. Navigate to csi-unity/dell-csi-hem-installer folder and execute this command:
    ```bash
    
    ./csi-install.sh --namespace unity --values ./myvalues.yaml --upgrade
    ```
 
-### Using Operator
-
-**Notes:**
-1. While upgrading the driver via operator, replicas count in sample CR yaml can be at most one less than the number of worker nodes.
-2. Upgrading the Operator does not upgrade the CSI Driver.
-
-To upgrade the driver:   
+### Upgrade using Dell CSM Operator:
+**Note:**
+Upgrading the Operator does not upgrade the CSI Driver.
 
-1. Please upgrade the Dell CSI Operator by following [here](./../operator).
-2. Once the operator is upgraded, to upgrade the driver, refer [here](./../../../installation/operator/#update-csi-drivers).
+1. Please upgrade the Dell CSM Operator by following [here](../../../../deployment/csmoperator/#to-upgrade-dell-csm-operator-perform-the-following-steps)
```

Comment:
1. Upgrade the Dell ...

(remove "Please")

Reply:
Done the changes. Removed the word "please" before instruction.